inbc-program/README.md

```diff
@@ -100,6 +100,11 @@ System update flow can be broken into two parts:
 1.  Pre-reboot: The pre-boot part is when a system update is triggered.
 2.  Post-reboot: The post-boot checks the health of critical manageability services and takes corrective action.
 
+SOTA on Ubuntu is supported in 3 modes:
+1. Update mode - SOTA in this mode will perform the software update.
+2. No download mode - SOTA in this mode will retrieve packages and install them.
+3. Download only mode - SOTA in this mode will only retrieve packages and it will not unpack them or install them.
```

Comment:
'SOTA in this mode' doesn't need to be stated.  That's implied.  Direct statements are better.  'Performs the software update'. 'Retrieves and installs packages', 'Retrieves the packages only (will not unpack or install).